src/Dhadgar.Scope/src/components/layout/Sidebar.tsx

```
@@ -1,105 +1,105 @@
+            </svg>
+            Dependency Map
+          </a>
+
+          <TextField
+            value={filter}
+            onChange={(e) => setFilter(e.target.value)}
+            placeholder="Search sections..."
+            clearable
+            onClear={() => setFilter("")}
+            icon={
+              <svg className="h-4 w-4" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)